src/app/Pages/Blog.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react'
+import {
+   PageSection, Bullseye, Text, TextContent, TextVariants,
+  Card, CardTitle, CardBody, CardFooter
+} from '@patternfly/react-core'
+import { useDocumentTitle } from '@app/utils/useDocumentTitle'
+import Footer from '@app/components/Footer'
+import post11png from '@app/bgimages/post1-1.png'
+
+const Blog: React.FunctionComponent<{title: string}> = ({title}) => {
+
+  useDocumentTitle(title)
+  return (
+    <div>
+      <PageSection isFilled={true}>
+      <Bullseye>
+        <Card
+          style={{
+            width: 800,
+            display: 'flex',
+            justifyContent: 'center',
+          }}>
+          <CardTitle>
+            <p>The new Cloud Image Directory is now available</p>
+            <Text component={TextVariants.small}>Author: The Cloud Experience Team</Text>
+          </CardTitle>
+          <CardBody isFilled={false}>
+            <Bullseye>
+              <img src={post11png}
+              style={{
+                display: 'flex',
+                justifyContent: 'center',
+              }}></img>
+            </Bullseye>
+            <TextContent>
+              <Text component={TextVariants.p}>
+                Our Cloud Experience team is excited to announce the availability of the first version of our Cloud Image Directory.
+                This tool is in an experimental stage, however we have decided to release this early access version to our community
```

Comment:
Please rewrite as follows:

Though this tool is in an experimental stage, we are releasing this early access version to our community so we stay true to our open culture and engineering principles. We invite you to join us in this journey.